<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetGenericPropertiesHead</class>
 <widget class="QWidget" name="WidgetGenericPropertiesHead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <widget class="QLabel" name="IdentifierLabel">
     <property name="text">
      <string>Identifier</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="colorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="IdentifierInfield"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="AuthorLabel">
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelPhase">
     <property name="text">
      <string>Phase</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QIntegerEdit" name="IndexInfield"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="DescInfield">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Enter description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ColorPicker" name="colorPicker" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="IndexLabel">
     <property name="text">
      <string>Index</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="NameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="AuthorInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="DescriptionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="NameInfield"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelDiscipline">
     <property name="text">
      <string>Discipline</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboDiscipline">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboPhase"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIntegerEdit</class>
   <extends>QLineEdit</extends>
   <header>qintegeredit.h</header>
  </customwidget>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header location="global">colorpicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>AuthorInfield</tabstop>
  <tabstop>comboDiscipline</tabstop>
  <tabstop>IndexInfield</tabstop>
  <tabstop>IdentifierInfield</tabstop>
  <tabstop>NameInfield</tabstop>
  <tabstop>DescInfield</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
